recipes-sota/garage-sign/garage-sign.bb

```diff
@@ -6,14 +6,14 @@ LICENSE = "CLOSED"
 LIC_FILES_CHKSUM = "file://${S}/docs/LICENSE;md5=3025e77db7bd3f1d616b3ffd11d54c94"
 DEPENDS = ""
 
-PV = "0.2.0-29-gf6f095a"
+PV = "0.2.0-35-g0544c33"
```

Comment:
Any reason to not use the brand-new version 37?

Reply:
None, I just didn't notice that it updated again. We can bump it from time to time.